tests/lang/s01_basics/s01_comments/t1_comments.nim

```diff
@@ -87,4 +87,34 @@ yeaaaaaaah
   ]##
 echo baz
 
-echo 13
\ No newline at end of file
+echo 13
+
+proc doStuff = 
+  ## Doc comments are valid ast so procs with them are valid!
```

Comment:
So is this implicitly a discard?

Reply:
Discard isnt unique any ast inside a block will make it valid, discard is another no op.